experimenter/experimenter/experiments/changelog_utils.py

```diff
@@ -87,3 +91,60 @@ def generate_nimbus_changelog(experiment, changed_by, message, changed_on=None):
         message=message,
         changed_on=changed_on,
     )
+
+
+def get_formatted_change_object(field_name, field_diff, changelog, timestamp):
+    event = NimbusConstants.ChangeEvent.find_enum_by_key(field_name.upper())
+    field_display_name = (
+        event.display_name
+        if event.value != NimbusConstants.ChangeEvent.TRIVIAL.value
+        else field_name
+    )
+
+    old_value = field_diff["old_value"]
+    new_value = field_diff["new_value"]
+
+    if event.value == "DATE_TIME":
+        if old_value is not None:
+            old_value = old_value.strftime("%B %d, %Y")
+        new_value = new_value.strftime("%B %d, %Y")
```

Comment:
does this need to use the timezone stuff like your other patch?

Reply:
Oh, this is just to format the date time change values to make them more readable. The other one is for the timestamps of the changes to include the timezone as well.